code-challenges.test.js

```diff
@@ -25,32 +34,114 @@
 // b) Create the function that makes the test pass.
 
 
+//input: number
+//output: array of numbers
+
+//result will always begin [1,1,2]
+// result array will be the length of the input
+//create an inital array [1,1,2]
+//using for loop and conditional, push current value into initial array
+//current value is sum of two previous values
+
+
+const fibonacci = (num) => {
+  var initial = [1, 1, 2]
+for (var i = 2; i < initial.length; i++) {
+    var currentValue = initial[i - 1] + initial[i - 2]
+    initial.push(currentValue)
+}
+return initial
+}
+
+//getting error: Fatal error in , line 0
+//# Fatal JavaScript invalid size error 184071938
+//error Command failed with signal "SIGTRAP"
+
+
+
+
 // --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
-// Expected output: [-9, 7, 9, 199]
+describe("oddOrdered", () => {
+  it("returns a new array of only odd numbers sorted from least to greatest", () => {
+    expect(oddOrdered(fullArr1)).toEqual([-9, 7, 9, 199])
+    expect(oddOrdered(fullArr2)).toEqual([-823, 7, 23])
+  })
+})
+
 
+var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+// // Expected output: [-9, 7, 9, 199]
+//
 var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
 
 // b) Create the function that makes the test pass.
 
+//check each value and return only numbers using filter
+//use filter again to have an array of only odd numbers
+//use .sort to put the numbers in order from least to greatest
+//possible refactor to filter two things at once?
+
+const oddOrdered = (fullArr) => {
+  var nums = fullArr.filter(value => (typeof value === 'number') && (value % 2 !== 0))
+  // var odds = nums.filter(number => number % 2 !== 0)
+
+  var sorted = nums.sort(function (a, b) {
+    return a - b;
+  });
+  return sorted
+}
+
```

Comment:
Nice job. As a refactor you can use an arrow in your sort function and shorten it up a bit.